src/systemdhook.c

```diff
@@ -18,6 +18,7 @@
 #include <linux/limits.h>
 #include <selinux/selinux.h>
 #include <yajl/yajl_tree.h>
+#include <stdbool.h>
```

Comment:
@mrunalp OK to use this library?  It require C99 and just wanted to double check that it was ok to rely on that.